bot/cogs/antispam.py

```diff
@@ -34,7 +34,8 @@
     'links': rules.apply_links,
     'mentions': rules.apply_mentions,
     'newlines': rules.apply_newlines,
-    'role_mentions': rules.apply_role_mentions
+    'role_mentions': rules.apply_role_mentions,
+    'everyone_ping': rules.apply_everyone_ping,
```

Comment:
line 38: at the end im pretty sure there doesn't need to be a comma, probably just nit picking here :)

Reply:
This diff is a good example of how the trailing comma is useful. Because line `37` did not have the trailing comma, it had to be changed in order to add another element.

Reply:
Yep, leave this trailing comma in - in fact, it's encouraged by PEP8 as well.

Reply:
Alright, i was just thinking it might not need to be there, im just someone who works in environments where using as least space as possible is the goal. :)